api/routers/edr.py

```diff
@@ -56,17 +63,27 @@
 async def get_locations(
     bbox: Annotated[str | None, Query(example="5.0,52.0,6.0,52.1")] = None
 ) -> EDRFeatureCollection:  # Hack to use string
+    # TODO: Remove duplication between `/collections` endpoint
+    # Get all unique parameters
+    ts_request = dstore.GetTSAGRequest(attrs=["parameter_name", "standard_name", "unit", "level", "period", "function"])
+    ts_response = await get_ts_ag_request(ts_request)
+    # logger.info(ts_response.ByteSize())
+
+    # Sadly, this is a different parameter as in the /locations endpoint, due to an error in the EDR spec
+    # See: https://github.com/opengeospatial/ogcapi-environmental-data-retrieval/issues/427
+    all_parameters: Dict[str, Parameter] = {}
+
+    for group in ts_response.groups:
+        ts = group.combo
+        parameter = make_parameter(ts)
+        all_parameters[ts.parameter_name] = parameter
```

Comment:
Is it possible that there is a duplicate parameter name? If so should then an exception be raised that it already exists?

Reply:
Yes, that is probably the right way/place to do it.

Reply:
I don't think it is possible. `make_parameter` creates the definition from variables used for parameter_name.